src/confd/bin/gen-interfaces

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+# Generates ietf-intefaces + ietf-ip settings for all detected interfaces
+# Interfaces must be of type Ethernet (copper/sfp), not be "internal" DSA
+# ports, and must have a "parentbus", i.e., be physical ports.
+#
+# Usage: gen-interfaces [-4bd]
+#
+#   -b brN  Bridged mode, set all interfaces as ports in brN
+#   -4      Generic option for both modes, enables ZeroConf IPv4
+#   -d      Generic option for both modes, enables DHCPv4 client
```

Comment:
These options seem to only apply to the bridged mode, no?

Reply:
Ah yes. Will add a note of this in the refactor task to support the CRB.